app/presenters/idv/in_person/ready_to_verify_presenter.rb

```diff
@@ -0,0 +1,86 @@
+require 'barby'
+require 'barby/barcode/code_128'
+require 'barby/outputter/png_outputter'
+
+module Idv
+  module InPerson
+    class ReadyToVerifyPresenter
+      attr_reader :enrollment
+      delegate :enrollment_code, to: :enrollment
+
+      def initialize(enrollment:)
+        @enrollment = enrollment
+      end
+
+      def barcode_data_url
+        "data:image/png;base64,#{Base64.strict_encode64(barcode_image_data)}"
+      end
+
+      def formatted_due_date
+        due_date.strftime(I18n.t('time.formats.event_date'))
+      end
+
+      def formatted_enrollment_code
+        enrollment_code.gsub(/(\d{4})/, '\1-').chomp('-')
+      end
+
+      def selected_location_details
+        # WILLFIX: After LG-6708, delegate this to enrollment.
+        {
+          name: 'BALTIMORE — Post Office™',
+          streetAddress: '900 E FAYETTE ST RM 118',
```

Comment:
Question: Will we want to reformat these keys to match snake_case conventions?

As-is, this matches what we receive from USPS ([example](https://github.com/18F/identity-idp/blob/a294bd6096b54a400be519baa9c7a2498aca5b7a/spec/fixtures/usps_ipp_responses/request_facilities_response.json#L3-L24)).

Reply:
Yes, we will most likely be storing the address data in a custom format; that's being worked out in LG-6089 & LG-6872, and I've added notes to those tickets to make sure to update this page when it's established.